libgraph/include/katana/GraphTopology.h

```diff
@@ -285,6 +285,13 @@ class KATANA_EXPORT EdgeShuffleTopology : public GraphTopology {
            (kind == edge_sort_state());
   }
 
+  bool has_edges_sorted_by_property(
```

Comment:
I don't think you need the first parameter as it should always be "EdgesSortedByProperty" (or whatever the name of enum value you added in RDGTopology.h)